JS/script.js

```diff
@@ -0,0 +1,5 @@
+const check = document.querySelector('.h1');
+
+check.addEventListener('click', () => {
+
+});
```

Comment:
Why don't we create the section "Featured speakers"  dynamically in JavaScript? 🤔 
![Screenshot (301)](https://user-images.githubusercontent.com/66112760/141965300-a12fd89a-bbd1-465d-8587-cf2c5972cce8.png)

Kindly use a JavaScript variable with the data about the speakers and use it when the page loads to create the HTML for this section dynamically. 👍